kinect_bridge2/KinectDepthImageInfo depthInfo

uint16 height_
uint16 width_
uint8 num_channels_
uint8 pixel_depth_
string encoding_